confirm/confirm.go

```diff
@@ -33,3 +33,28 @@ func Ask(s string) bool {
 		}
 	}
 }
+
+// AskStrict asks the user for confirmation. A user must type the expected confirmation text and
+// then press enter.
+func AskStrict(s string, c string) bool {
+	reader := bufio.NewReader(os.Stdin)
+
+	fmt.Printf("%s: ", color.YellowString(s))
+
+	for {
```

Comment:
I think it would work the same way without the for loop, and removing it would mean having less code!